pkg/opentelemetry/output.go

```diff
@@ -36,47 +52,198 @@ func New(p output.Params) (*Output, error) {
 
 // Description returns a human-readable description of the output that will be shown in `k6 run`
 func (o *Output) Description() string {
-	return "OpenTelemetry: " + o.config.Address
+	return "opentelemetry: " + o.config.String()
 }
 
 // Stop flushes all remaining metrics and finalizes the test run
 func (o *Output) Stop() error {
 	o.logger.Debug("Stopping...")
 	defer o.logger.Debug("Stopped!")
+
+	if err := o.meterProvider.Shutdown(context.Background()); err != nil {
+		o.logger.WithError(err).Error("Error shutting down metric provider")
+	}
+
 	o.periodicFlusher.Stop()
+
 	return nil
 }
 
 // Start performs initialization tasks prior to Engine using the output
 func (o *Output) Start() error {
-	o.logger.Debug("Starting...")
+	o.logger.Debug("Starting output...")
+
+	ctx := context.Background()
+	// TODO: support different exporters (e.g. OTLP/HTTP)
+	exp, err := otlpmetricgrpc.New(
+		ctx,
+		otlpmetricgrpc.WithInsecure(),
+		otlpmetricgrpc.WithEndpoint(o.config.GRPCReceiverEndpoint),
+	)
+	if err != nil {
+		return fmt.Errorf("failed to create OTEL exporter: %w", err)
```

Comment:
```suggestion
		return fmt.Errorf("failed to create OpenTelemetry exporter: %w", err)
```

Should we use the entire name in the error messages instead of the acronym?